shared/js/contacts/search.js

```diff
@@ -87,49 +87,52 @@
       return;
```

Comment:
We will need to return Promise.resolve here, as now we are expecting to return a promise.